src/components/LayerUI.tsx

```diff
@@ -354,13 +345,7 @@ const LayerUI = ({
             )}
           >
             <UserList collaborators={appState.collaborators} />
-            {onCollabButtonClick && (
-              <CollabButton
-                isCollaborating={isCollaborating}
-                collaboratorCount={appState.collaborators.size}
-                onClick={onCollabButtonClick}
-              />
-            )}
+            {childrenComponents.LiveCollaboration}
```

Comment:
I'm thinking we might wanna export a slot instead (`<SlotTopRight>` or something), and reuse for `<UserList>`.

Reply:
But let's leave it for later. It's fine if we break (remove) the `LiveCollaboration` compo in the next PR because I bet few are using it.

Reply:
Though I'll need to do that in fork in the meantime

Reply:
didn't get you, why would we remove the `Live Collaboration` comp ?

Reply:
as a top-level component I meant